cms/static/sass/views/_outline.scss

```diff
@@ -183,10 +182,8 @@
   .course-status {
     margin-bottom: $baseline;
 
-    .status-release,
```

Comment:
It looks like this is used elsewhere than just the files you touched below: https://github.com/edx/edx-platform/search?q=status-release&unscoped_q=status-release (underscore file as well as in a different scss file). Can we sanity check the deletion here won't break other places (not sure how/if those files are leveraged).

Reply:
Just for record-keeping: we checked, and this does apply to the scheduled release section on a section/subsection and affects, for example, the transition in the opacity, so I will add this back in.